packages/vscode-extension/package.json

```diff
@@ -309,8 +309,12 @@
                   }
                 }
               },
+              "packageManager": {
+                "type": "string",
```

Comment:
Couldn't that be an enum?

Reply:
it is not supported https://code.visualstudio.com/api/references/contribution-points